integration-cli/docker_cli_build_test.go

```diff
@@ -1568,7 +1569,18 @@ func TestBuildAddBadLinks(t *testing.T) {
 	}
 	defer os.RemoveAll(tempDir)
 
-	symlinkTarget := fmt.Sprintf("/../../../../../../../../../../../..%s", tempDir)
+	var symlinkTarget string
```

Comment:
@LK4D4 great idea but as far as I can tell, no:

    filepath.Join("/", "foo", "bar") → "\\foo\bar" (not a valid URL on windows)

since we need the volume letter (`c:`) at the beginning I might give a full URL of where we're running:

    filepath.Join(os.Getwd(), "..","..","..","..","..","..","..","..", "foo") → "c:\foo"

is still not what I want since filepath.Join clears the path from unnecessary stuff like "..".